sparkle/src/components/Breadcrumbs.tsx

```diff
@@ -105,10 +105,7 @@ export function Breadcrumbs({ items }: BreadcrumbProps) {
 
 function truncateWithTooltip(text: string, length: number) {
   return text.length > length ? (
-    <Tooltip
-      label={text}
-      position="below"
-    >{`${text.substring(0, length - 1)}…`}</Tooltip>
+    <Tooltip trigger={`${text.substring(0, length - 1)}…`} label={text} />
```

Comment:
`nit` We should use css for the ellipsis: `text-ellipsis`.

Reply:
We won't have control over the truncation length though?